modules/core/04-channel/v2/keeper/grpc_query.go

```diff
@@ -136,19 +60,15 @@ func (q *queryServer) PacketCommitment(ctx context.Context, req *types.QueryPack
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
-	if err := host.ChannelIdentifierValidator(req.ChannelId); err != nil {
+	if err := host.ClientIdentifierValidator(req.ClientId); err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
 	if req.Sequence == 0 {
 		return nil, status.Error(codes.InvalidArgument, "packet sequence cannot be 0")
 	}
 
-	if !q.HasChannel(ctx, req.ChannelId) {
```

Comment:
This was a pain to find the analog of especially in a world with aliasing. Just decided to remove it since if client (or aliased client) doesn't exist, none of these other auxilliary state will exist either